feat: add `blas/ext/base/ndarray/ddiff` (#12870)
stdlib-js/stdlib: lib/node_modules/@stdlib/blas/ext/base/ndarray/ddiff/docs/repl.txt

{{alias}}( arrays )
    Calculates the k-th discrete forward difference of a one-dimensional

>     double-precision floating-point ndarray.

Reviewer: ```suggestion
    Calculates the k-th discrete forward difference of a one-dimensional double-
    precision floating-point ndarray.
```

    The output ndarray must have `N + N1 + N2 - k` elements, where `N` is the

>     number of indexed elements in `x`, `N1` is the number of elements to

Reviewer: ```suggestion
    number of elements in the input ndarray, `N1` is the number of elements to
```

The use of "indexed" only applies to strided arrays. You'll need to rewrap this paragraph.

Reviewer: Also "x" doesn't have any meaning here.

    prepend, `N2` is the number of elements to append, and `k` is the number

>     of differences.

Reviewer: ```suggestion
    of times to compute differences.
```

>     of times to compute differences.

Reviewer: ```suggestion
    of times to recursively compute differences.
```
